community-spotlight-heart-transplant-tracker.mdx

```diff
@@ -0,0 +1,88 @@
+
+Daniel's application provides clear insights into his son's position on the transplant list, and the visual representation of the data makes it much easier to understand daily changes and trends over time. It's also empowered Daniel and his to have more informed and meaningful conversations with their transplant coordinator because they can quickly identify any significant movements on the list, and discuss potential opportunities or raise concerns.
+
+The application features several key components:
+
+- **Daily updates:** Each morning, Daniel and his wife can check the app to see the latest data on the transplant list. The app highlights any changes from the previous day, providing immediate insight into whether their son's position has improved or worsened.
+
+- **Historical data:** The app stores historical data, allowing Daniel and his wife to track trends over time. This historical perspective helps them understand how the list evolves and to anticipate future changes.
+
+![Historical data easily accessible at a glance](/images/heart-transplant-tracker/historical-data-crop.png)
+
+- **Visual graphs:** Visual graphs and charts have helped to make the complex data more digestible so that Daniel can quickly grasp the current state of the list and identify any notable patterns or anomalies.
+
+![Visual graphs showing the data](/images/heart-transplant-tracker/visual-graphs-crop.png)
+
+Daniel hopes that by sharing his story, it may help others going through similar experiences.
+For more details on Daniel's project, you can visit the [Heart Transplant List Tracker app](https://heart-transplant.netlify.app/) and his Github repo at https://github.com/danreale/transplant. If you'd like to get in touch with Daniel, you can reach him on [LinkedIn](https://www.linkedin.com/in/danreale/).
+
+## Feedback and feature requests
+
+We asked Daniel what Xata features he found most useful and what he feels may be missing today. Here's what he shared:
+
+- **Intuitive user interface:** the simplicity and visual ease that Xata provides because he prefers GUI-based interactions over scripting, and it's helped reduce the complexity around database management.
+- **Flexible language queries:** the ability to use both Xata's domain-specific language and SQL for building queries has been very useful.
+
+> I would definitely recommend Xata to other developers. It's been a fantastic tool for my side projects and has made a big difference in how I manage data.\
+> _Daniel Reale - QA Engineer_
+
+Areas for improvement include:
+
+- **Enhanced domain language features:** Daniel noted the need for more advanced functionalities for handling distinct records and limiting functions to avoid having to use standard SQL for certain tasks.
+- **Usability issues:** He experienced flashing issues with the table view after prolonged use, likely due to potential session handling or memory management problems, but we have not yet been able to replicate the issue.
```

Comment:
```suggestion
- **Usability issues:** More robust session handling because he recently experienced flashing issues with the table view after prolonged use, likely due to potential session handling or memory management problems, but we have not yet been able to replicate the issue.
```

Reply:
@alexfrancoeur is this what you had in mind?

Reply:
I was thinking of swapping "Usability issues" with "Robust session handling" or something along those lines. What do you think?

Reply:
Oh I see, sorry I misunderstood. Yes that works. I'll make the change.